tests/test_lightgbm.py

```diff
@@ -34,7 +34,9 @@ def test_cpu(self):
 
         self.assertEqual(1, gbm.best_iteration)
 
+    # TODO(b/381256047): Colab needs to install GPU-enabled lightgbm.
```

Comment:
Do you consider this a launch blocker for this?

Reply:
re-added it in #1451

Colab is also working on adding it to the base so we can re-drop once its out in their image